src/main/java/com/elixr/poc/Constant/ApplicationConstants.java

```diff
@@ -0,0 +1,9 @@
+package com.elixr.poc.Constant;
+
+/* this constant file holds all the error messages for all the end points */
```

Comment:
Recommended comment format is /** */. Modify it in all files

Reply:
Updated.